CalculatorIdo/Script.js

```diff
@@ -0,0 +1,149 @@
+window.onload = () => {clearCalculator();}
+
+var currAns = '';
+var isResultShown = false;
+var isbracketOpen = false;
+
+clearCalculator = () => {document.calcForm.input.value = '';}
+
+del = () => {
+    if(isResultShown) {
+        isResultShown = false;
+        clearCalculator();
+    }
+
+    let currString = document.calcForm.input.value;
+
+    // Deleted the whole word if its a word
+    if(/^[a-zA-Z]+$/.test(currString.charAt(currString.length - 1))) {
+        document.calcForm.input.value = currString.substring(0, currString.length - 1);
```

Comment:
I would prefer you'd create two classes, one for the UI and one for the logic. rather than mixing them together. (this comment is for the whole project and just this line)